.github/workflows/manual-deploy-environment.yml

```
@@ -55,17 +55,17 @@ jobs:
       FLEEK_PROJECT_ID: ${{ secrets.FLEEK_PROJECT_ID }}
     needs: [build-and-test-all, cypress-run]
     steps:
-      - name: Checkout
-        uses: actions/checkout@v3
+    - name: Checkout
+      uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)